Home.md

```diff
@@ -17,3 +17,73 @@ Use the [`#WindowsCommunityToolkit`](https://twitter.com/search?q=%23WindowsComm
+
+<h1 align="center">Fiona</h1>
+
+<table>
+  <tr>
+    <td><img src=https://github.com/Kyaa-dost/WindowsCommunityToolkit-wiki/blob/App-Gallery/images/Fiona1.png height=350></td>
+    <td><img src=https://github.com/Kyaa-dost/WindowsCommunityToolkit-wiki/blob/App-Gallery/images/Fiona2.png height=350></td>
+  </tr>
+  <tr>
+    <td><img src=https://github.com/Kyaa-dost/WindowsCommunityToolkit-wiki/blob/App-Gallery/images/Fiona4.png height=350></td>
+    <td><img src=https://github.com/Kyaa-dost/WindowsCommunityToolkit-wiki/blob/App-Gallery/images/Fiona3.png height=350></td>
+  </tr>
+</table>
+
+>Creator: [PaoloM](https://github.com/PaoloM)
+
+<p align="left">Fiona was started using the WCT Template, and it's using a bunch of the helpers and services provided with that. Plus, as mentioned, the toolkit MVVM.</p>
+ 
+ > You can find out more information about **Fiona** [here](https://github.com/PaoloM/Fiona)
+
+<h1 align="center">Shrestha Files Pro</h1>
+
+![Shrestha](https://github.com/Kyaa-dost/WindowsCommunityToolkit-wiki/blob/App-Gallery/images/Shrestha.gif)
+
+Shrestha Files is a modern and minimalist dual pane file manager with tabs and color themes support.
+
+- It uses GridSplitter in dual pane mode to resize two panes.
+- It uses Guard to validate arguments.
+- It uses MVVM toolkit and is gradually moving to MVVM-ify more! All settings pages and navigation bar edit page mostly follow MVVM pattern.    
+- It uses EnumValuesExtensions since the last version to load enum values in SplitButton options for view type and sort type.
+
+> Here's a quick overview of: [Shretha Files](https://youtu.be/-LBXu0lzh_Y "https://youtu.be/-LBXu0lzh_Y"). Find the App [here](https://www.microsoft.com/p/shrestha-files-pro-a-modern-dual-panel-file-manager-with-tabs-and-colors/9npnffsv2hqm)
+
+<h1 align="center">Yugen Mosaic</h1>
+
+![YugenMosaic](https://github.com/Kyaa-dost/WindowsCommunityToolkit-wiki/blob/App-Gallery/images/YugenMosaic.png) 
+![YugenMosaic2](https://github.com/Kyaa-dost/WindowsCommunityToolkit-wiki/blob/App-Gallery/images/YugenMosaic2.png)
+
+>Creator: [PandaSharp](https://github.com/Panda-Sharp)
+
+Yugen Mosaic is a free app that creates digital art pictures based on your own images. With it, you can use an unlimited amount of images, to create a photo mosaic which is an image composed of many tiled photos. Yugen is an awareness of the universe that triggers emotional responses too deep and powerful for words, and that is just what your art will do with a little help from Yugen Mosaic!
+
+Find the App in [Microsoft Store](https://www.microsoft.com/en-us/p/yugen-mosaic/9pf0s24cx0d4). More information regarding this [project](https://github.com/Panda-Sharp/Yugen.Mosaic)
+
+<h1 align="center">OneLocker and Legere</h1>
+
+![Onelocker-legere](https://github.com/Kyaa-dost/WindowsCommunityToolkit-wiki/blob/App-Gallery/images/Onelocker-Legere.png)
+
+>Creator: [Sergio Pedri](https://github.com/Sergio0694)
```

Comment:
@Sergio0694 which one would you want to highlight here? Pick one. 😉

Reply:
I'm confused - is the issue just that the image has two apps together, or do we want to limit the gallery to one app per developer? Because if it's the latter then I guess I'd have to also remove my console/IDE further down in the page as well? Let me know, and if needed I can provide alternative assets as well or anything else 🙂

Reply:
@Sergio0694 As of now we would like to have one image so we don't clutter the page too much.

Reply:
@Sergio0694 let me know whichever project you want to be featured in the Gallery and please provide Github/project link of that as well. Thanks 😊

Reply:
Hey @Kyaa-dost, sorry for the delay! Got busy working on a PR for .NET 6 to support the MVVM Toolkit 😄

Between the two I guess I'd prefer to feature Legere, as it's the most recent app and it uses the Toolkit the most.
Here's an image just with that: [Legere](https://user-images.githubusercontent.com/10199417/122304755-05dc2900-cf06-11eb-96ab-1681b36677bb.png).
Here's the Store link: https://www.microsoft.com/store/apps/9PHJRVCSKVJZ.

And just in case you decided to allow more than one app per dev, then the second one for me would be my Console/IDE, which is also open source on GitHub, and heavily uses a lot of stuff from the Toolkit, especially .NET APIs such as the HighPerformance and Diagnostics packages and the MVVM Toolkit 🚀
Here's an updated image for that: [console/IDE](https://user-images.githubusercontent.com/10199417/122305064-869b2500-cf06-11eb-8c6a-d5b7b04eb614.png).
And here's the GitHub link: https://github.com/Sergio0694/Brainf_ckSharp.

Let me know if this is alright!

Reply:
Yeah, for now just wanted to limit one-per dev. I think eventually we'll try and better integrate with the UWP Community site, we're just kind of doing this as a temporary measure in the meantime.

Reply:
I see, sounds good! 😄